app/assets/javascripts/maps.js

```diff
@@ -0,0 +1,31 @@
+function displayMap(address) {
+    var addressObj = $.get( 'https://maps.googleapis.com/maps/api/geocode/json?address='+address);
+
+    var the_url = 'https://maps.googleapis.com/maps/api/geocode/json?address='+address
+
+    $.ajax({
+        url: the_url,
+        dataType: 'json',
+        type: 'get',
+        success: function(data) {
+            lat = data.results[0].geometry.location.lat;
+            lng = data.results[0].geometry.location.lng;
+
+            if(lat!=null && lng!= null){
+                var myLatlng = new google.maps.LatLng(lat,lng);
```

Comment:
'google' is not defined.<br>'lat' is not defined.<br>'lng' is not defined.<br>'myLatlng' is defined but never used.